tools/serverpod_cli/test_e2e/lib/src/keyword_search_in_stream.dart

```diff
@@ -33,6 +33,7 @@ class KeywordSearchInStream {
 
     _found = null;
 
+    await Future.delayed(const Duration(milliseconds: 250));
```

Comment:
For future reference - The delay was added here because on Linux we are not gauranteed that files are removed at the time `delete(..)` is called in dart.